modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -98,9 +98,9 @@ Creating a machine pool with taints is only possible if the cluster already has
 Alternatively, you can add the node labels and taints after you create the machine pool.
 ====
 
-ifdef::openshift-rosa[]
+ifdef::openshift-rosa,openshift-dedicated[]
 . Optional: Select additional custom security groups to use for nodes in this machine pool. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for _Security groups_ under _Additional resources_.
```

Comment:
```suggestion
. Optional: Select additional custom security groups to use for nodes in this machine pool. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for security groups in the "Additional resources" section.
```
- See IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element
- No highlighting is needed for the phrase "security groups" in this sense. If you were referring to a title in the documentation, you could use double-quotation marks.
- I updated the highlighting of "Additional resources" from italics to double-quotation marks per the IBM Style "Highlighting" topic.
- From [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#grammar): "Include the word _that_ in clauses unless it results in writing that sounds too formal or stilted. The use of the conjunction _that_, although technically optional in some sentences, is never wrong and makes the sentence easier to translate and clearer for readers whose primary language is not English."

Reply:
amended and found it also in modules/creating-a-machine-pool-cli.adoc